pkg/download/work_queue.go

```diff
@@ -1,26 +1,52 @@
 package download
 
-// workQueue takes work items and executes them serially, in strict FIFO order
+// workQueue takes work items and executes them serially, with n concurrent
+// workers.  It allows for a simple high/low priority split between work.  We
```

Comment:
Maybe I'm being slow, but what does "with n concurrent workers" mean here?

Reply:
it means I forgot to put a for loop in the code 🤦 I will also rewrite the comment to make more sense